public/js/profile.js

```diff
@@ -11,6 +11,7 @@ $(function() {
   loadIssues();
   $('#check').click(checkPRs);
   $('#share_info').change(saveShareInfo);
+  toggleStatus();
```

Comment:
Do we need this initial call to `toggleStatus`?

Reply:
No we don't actually need this now. I made an the following update to profile.css on my local version which means we shouldn't need an initial call to `toggleStatus`

```
#results {
  display: none;
}